src/meta/src/controller/streaming_job.rs

```diff
@@ -473,6 +473,15 @@ impl CatalogController {
         if let Some(source_id) = associated_source_id {
             Object::delete_by_id(source_id).exec(&txn).await?;
         }
+
+        for tx in inner.table_id_to_tx.remove(&job_id).into_iter().flatten() {
```

Comment:
This seems to suggest there's more than one tx per table id. Should it be `if let Some(tx) = ...` instead?